app/views/tag/_tags_popover.html.erb

```diff
@@ -1,6 +1,7 @@
 <div class="main_div">
   <p class="small_heading"> Follow related tags: </p>
   <%= form_with remote: true, url: '/subscribe/multiple/tag' do |f| %>
+  <% unless tags.nil? %>
     <% tags.each do |tag| %>
```

Comment:
Hi, @shubhi2000. I know this PR has been closed, but there's a shorter way to check for existence of tags:

```rb
<% tags && tags.each do |tag| %>
```
The first `tags` will return false if `tags` is nil, and true if it isn't. The `&&` operator tells Ruby to execute the command on the right only if the command on the left returns `true`

Reply:
@madeofhuman  I'll take care of it from next time onwards. Thank you so much :)